services/app-web/src/components/Banner/SubmissionUnlockedBanner/SubmissionUnlockedBanner.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styles from './SubmissionUnlockedBanner.module.scss'
+import { Alert } from '@trussworks/react-uswds'
+import dayjs from 'dayjs'
+import timezone from 'dayjs/plugin/timezone.js'
+import advancedFormat from 'dayjs/plugin/advancedFormat.js'
+
+dayjs.extend(advancedFormat)
```

Comment:
I don’t love seeing these global calls here. It looks like we are calling `dayjs.extend(isLeapYear)` in RateDetailSchmea as well. 

@haworku is this best practice? could we be requiring all our dayjs extensions somewhere more central? Or do you need to invoke dayjs.extend in every file where you use one of the extensions?

Reply:
@macrael For sure agreed.  Much better to isolate in a lib or by applying it across the module once. We can make a `dateHelpers/dayjs.ts` placed at the root of app-web src. I think if we import into `index.tsx` and extend there it should work across the app (even when importing from the original `dayjs` but I'd have to check). If not we can make it a dayjs internal lib we reference to access this dependency.

Reply:
@macrael @haworku  I put `dayjs` import and extensions in `dateHelpers/` directory to isolate and centralize the extensions. I also applied the change for `RateDetailSchmea`. Let me know what you think of this approach.